a-tier-bonding/src/components/header/header.js

```diff
@@ -5,6 +5,7 @@ import ButtonGroup from '@mui/material/ButtonGroup';
 import { Link } from "react-router-dom";
 import './header.css'
 import earthLogo from "../../assets/earth-logo.png"
```

Comment:
We should be able to remove this import now

Reply:
No it's still being used in header